tasks/framework-tools/lib/framework.mjs

```diff
@@ -152,7 +151,7 @@ export function cleanPackages(packages = frameworkPkgJsonFiles()) {
     {
       shell: true,
       stdio: 'inherit',
-      cwd: path.resolve(__dirname, '../../../../'),
+      cwd: path.resolve(__dirname, '../../../'),
```

Comment:
This file's location hasn't changed, should the path here have changed?

Reply:
I did move it actually; the filepath used to be `tasks/framework-tools/src/lib/framework.mjs`.